src/tools/tests/goldens/generated-schemas.jvm.kt

```diff
@@ -16,459 +16,458 @@ import arcs.core.entity.SchemaRegistry
 import arcs.core.entity.toPrimitiveValue
 import arcs.sdk.*
 
-@Suppress("UNCHECKED_CAST")
-class GoldInternal1(
-    val_: String = "",
-    entityId: String? = null,
-    expirationTimestamp:  Long = RawEntity.UNINITIALIZED_TIMESTAMP,
-    creationTimestamp: Long = RawEntity.UNINITIALIZED_TIMESTAMP
-) : EntityBase("GoldInternal1", SCHEMA, entityId, expirationTimestamp, creationTimestamp) {
-
-    var val_: String
+typealias Gold_Data_Ref = AbstractGold.GoldInternal1
+typealias Gold_Alias = AbstractGold.GoldInternal1
+typealias Gold_AllPeople = AbstractGold.Gold_AllPeople
+typealias Gold_QCollection = AbstractGold.Gold_QCollection
+typealias Gold_Collection = AbstractGold.Gold_Collection
+typealias Gold_Data = AbstractGold.Gold_Data
+abstract class AbstractGold : BaseParticle() {
```

Comment:
A blank line separating the class from the typealiases would be great

Reply:
Added!